src/main/java/org/matsim/run/MetropoleRuhrScenario.java

```diff
@@ -263,6 +263,20 @@ protected Config prepareConfig(Config config) {
 		// snz activity types that are always the same, Differentiated by typical duration
 		SnzActivities.addScoringParams(config);
 
+
+		//ride scoring params
+
+		// alpha can be calibrated
+		double alpha = 2.0;
+		//gamma must stay one
+		double gamma = 1.0;
+		double monetaryDistanceRateRide =  config.scoring().getOrCreateModeParams(TransportMode.car).getMonetaryDistanceRate() * alpha;
+		double marginalUtilityOfTravelingRide = (alpha + gamma) * config.scoring().getPerforming_utils_hr() + config.scoring().getOrCreateModeParams(TransportMode.car).getMarginalUtilityOfTraveling() * (1.0 + alpha) ;
```

Comment:
There is a minus missing in front of config.scoring().getPerforming_utils_hr(). While travelling we miss out on the opportunity to perform an activity and obtain beta_performing